content/SCALECLIReference/Storage/CLIDisk.md

```diff
@@ -11,7 +11,624 @@ tags:
+| 170 | Reserved_Block_Pct      | 100   | 100   | 10     |             | <dict> | <dict> |
+| 171 | Program_Fail_Count      | 100   | 100   | 0      |             | <dict> | <dict> |
+| 172 | Erase_Fail_Count        | 100   | 100   | 1      |             | <dict> | <dict> |
+| 173 | Avg_Block-Erase_Count   | 100   | 100   | 0      |             | <dict> | <dict> |
+| 174 | Unexpect_Power_Loss_Ct  | 100   | 100   | 0      |             | <dict> | <dict> |
+| 183 | SATA_Int_Downshift_Ct   | 100   | 100   | 0      |             | <dict> | <dict> |
+| 184 | End-to-End_Error        | 100   | 100   | 0      |             | <dict> | <dict> |
+| 187 | Reported_Uncorrect      | 100   | 100   | 0      |             | <dict> | <dict> |
+| 188 | Command_Timeout         | 100   | 100   | 0      |             | <dict> | <dict> |
+| 194 | Temperature_Celsius     | 59    | 48    | 0      |             | <dict> | <dict> |
+| 195 | Hardware_ECC_Recovered  | 100   | 100   | 0      |             | <dict> | <dict> |
+| 196 | Reallocated_Event_Count | 100   | 100   | 0      |             | <dict> | <dict> |
+| 197 | Current_Pending_Sector  | 100   | 100   | 0      |             | <dict> | <dict> |
+| 198 | Offline_Uncorrectable   | 100   | 100   | 0      |             | <dict> | <dict> |
+| 199 | UDMA_CRC_Error_Count    | 100   | 100   | 0      |             | <dict> | <dict> |
+| 202 | Percent_Lifetime_Remain | 100   | 100   | 1      |             | <dict> | <dict> |
+| 206 | Write_Error_Rate        | 100   | 100   | 0      |             | <dict> | <dict> |
+| 246 | Total_LBAs_Written      | 100   | 100   | 0      |             | <dict> | <dict> |
+| 247 | Host_Program_Page_Count | 100   | 100   | 0      |             | <dict> | <dict> |
+| 248 | Bckgnd_Program_Page_Cnt | 100   | 100   | 0      |             | <dict> | <dict> |
+| 180 | Unused_Rsvd_Blk_Cnt_Tot | 1     | 1     | 0      |             | <dict> | <dict> |
+| 210 | RAIN_Success_Recovered  | 100   | 100   | 0      |             | <dict> | <dict> |
++-----+-------------------------+-------+-------+--------+-------------+--------+--------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Temperature Command
+
+The `temperature` command returns the temperature of a specified disk.
+
+{{< expand "Checking Disk Temperature">}}
+
+#### Description
+The `temperature` command requires you to specify which disk you want to see the temperature for. You can also set the S.M.A.R.T. `powermode`, and retrieve previous temperatures using the `cache` option.
+Enter the command string, then press <kbd>Enter</kbd>.
+The command returns a temperature (in Celsius) when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>storage disk temperature name=<i>diskname</i> options={"cache":"<i>seconds</i>","powermode":"<i>MODE</i>"}<code>
+
+Where 
+* *diskname* is the name of a disk.
+* *seconds* is how far back in seconds you want to view the disk temperature. The default is null and returns the current temperature.
```

Comment:
the Null sentence goes with the description of the option. In this case `options` is the property and it has two required options `cache` and `powermode`. My comments above should reflect this but I didn't know how to use them when I reviewed that text.